docs/pages/ci.md

```diff
@@ -15,19 +15,21 @@ layout: default
 
 <details>
 <summary> 🟢 explanation</summary>
-We have many projects using github CI, it's both convenient and works well. It's become the standard. Also, note that it's free for public repos.
+We have many projects using GitHub CI and, it has good integration with GitHub itself, and is free for public repositories (with limited free monthly minutes for private repositories).
```

Comment:
Seeing as this is the first time we are using CI, should we define it?
```suggestion
We have many projects using GitHub continuous integration (CI) and, it has good integration with GitHub itself, and is free for public repositories (with limited free monthly minutes for private repositories).
```

Reply:
Would be nice if this page could be wrapped to 80 chars, in keeping with other pages

Reply:
I think if we want to enforce wrap to 80 it would be best to do via a pre-commit hook instead of having to manually edit?

Reply:
For wrapping I opened a new issue at https://github.com/UCL-ARC/python-tooling/issues/337